src/Schema/Column.php

```diff
@@ -227,7 +227,7 @@ public function setPlatformOption($name, $value)
     }
 
     /**
-     * @param string $value
+     * @param string|null $value
```

Comment:
This allow `$column1->setColumnDefinition($column2->getColumnDefinition())`
And this was the only setter which it wasn't possible.

Reply:
This looks good.